src/libraries/Directory.Build.targets

```diff
@@ -240,6 +237,22 @@
     <ExcludeFromPackage Condition="'$(TargetFramework)' == '$(NetFrameworkCurrent)' and '$(ExcludeCurrentFullFrameworkFromPackage)' == 'true'">true</ExcludeFromPackage>
   </PropertyGroup>
 
+  <!-- Adds System.Runtime.Versioning*Platform* annotation attributes to < .NET 5 builds -->
+  <Choose>
+    <When Condition="('$(IncludePlatformAttributes)' == 'true' or '$(IsWindowsSpecific)' == 'true') and ('$(TargetFrameworkIdentifier)' != '.NETCoreApp' or $(TargetFramework.StartsWith('netcoreapp')))">
+      <ItemGroup>
+        <Compile Include="$(CoreLibSharedDir)System\Runtime\Versioning\PlatformAttributes.cs" Link="System\Runtime\Versioning\PlatformAttributes.cs" />
+      </ItemGroup>
+    </When>
+  </Choose>
+
+  <!-- Adds MinimumOSPlatform attribute for Windows Specific libraries -->
+  <ItemGroup Condition="'$(IsWindowsSpecific)' == 'true' and '$(IsTestProject)' != 'true'">
```

Comment:
Can this attribute be part of ref assemblies? Can we instead change the condition to:

```xml
Condition="'$(IsWindowsSpecific)' == 'true' and '$(IsSourceProject)' == 'true'"
```

Reply:
Yes, it should. I was not sure when I started working on this task and I asked about this on Teams channel:

![obraz](https://user-images.githubusercontent.com/6011991/87717110-2eb4e500-c7b0-11ea-8eaf-fc237c64e683.png)

Reply:
Makes sense.